specification/draft/index.html

```diff
@@ -905,89 +905,95 @@ <h3>Verifiable Presentation of DSCSA Stakeholder Credentials</h3>
-            <td><b>aud</b></td>
+            <td><b>`aud`</b></td>
             <td>
-              Verifiable Presentation Audience. We might include PI_Verification in this field to represent that this VP
-              is used in this context
-              <br>
-              We might include ATP_DSCSA in this field to represent that this VP is used in this context
+              A digital wallet SHALL NOT include an `aud` field in the JWT payload. The recipient's DID of the JWT is
+              not known during presentation generation.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>iat</b></td>
+            <td><b>`iat`</b></td>
             <td>
-              Issuance Date of the VP in EPOCH format | Used for determining whether or not the VP is valid by including
-              the OCI validity time frame defined in Digital Wallet Conformance Criteria
+              A digital wallet SHALL include an `iat` field in the JWT payload. This field represents the UNIX timestamp
+              of the issuance date of the Verifiable Presentation.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>nbf</b></td>
+            <td><b>`nbf`</b></td>
             <td>
-              Issuance Date of the VC in EPOCH format
+              A digital wallet MAY include a `nbf` field in the JWT payload. This field represents the UNIX timestamp of
+              the date from when on the Verifiable Presentation is valid.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>exp</b></td>
+            <td><b>`exp`</b></td>
             <td>
-              Expiration Date of the VC in EPOCH format
+              A digital wallet SHALL include an `exp` field in the JWT payload. This field represents the UNIX timestamp
+              expiration date of the Verifiable Presentation and SHALL be five minutes in the future based on the
+              issuance date.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>vp</b></td>
+            <td><b>`nonce`</b></td>
             <td>
-              Contains the Verifiable Presentation
+              A digital wallet SHALL include a `nonce` field in the JWT payload. This field represents a random value
+              that is used to prevent replay attacks.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>vp.type</b></td>
+            <td><b>`vp`</b>
+            </td>
+            <td>
+              A digital wallet SHALL include a `vp` object in the JWT payload. This object contains further data of
+              the Verifiable Presentation.
+            </td>
```

Comment:
how would an implementer know what kind of  "further data" are required/allowed here?

Reply:
This is defined by the VC spec. It basically contains the rest of the data not being present in the form of iat, nbf, iss, ...